src/Microsoft.OData.Client/Serialization/HttpWebRequestMessage.cs

```diff
@@ -239,26 +316,19 @@ public override void SetHeader(string headerName, string headerValue)
         /// <returns>Stream to which the request payload needs to be written.</returns>
         public override Stream GetStream()
         {
-#if PORTABLELIB
-            // not supported in async environments. Another IODataRequestMessage which buffers the request should be used.
-            throw new NotSupportedException(Strings.ODataRequestMessage_GetStreamMethodNotSupported);
-#else
             if (this.inSendingRequest2Event)
             {
                 throw new NotSupportedException(Strings.ODataRequestMessage_GetStreamMethodNotSupported);
             }
-
-            return this.httpRequest.GetRequestStream();
-#endif
+            return _messageStream;

```

Comment:
```suggestion
            return this._messageStream;

```

Reply:
@marabooy  there is a review comment where someone suggested we remove this.{property} to _{property}